processor/metricstarttimeprocessor/README.md

```diff
@@ -50,3 +50,20 @@ Cons:
 * The True Reset point doesn't make sense semantically. It has a zero duration, but non-zero values.
 * Many backends reject points with equal start and end timestamps.
     * If the True Reset point is rejected, the next point will appear to have a very large rate.
+
+### Strategy: Subtract Initial Point
+
+The `subtract_initial_point` strategy handles missing start times for
+cumulative points by dropping the first point in a cumulative series,
+"subtracting" that point's value from subsequent points and using the initial
+point's timestamp as the start timestamp for subsequent points.
+
+Pros:
+
+* Cumulative semantics are preserved. This means that for a point with a given `[start, end]` interval, the cumulative value occurred in that interval.
+* Rates over resulting timeseries are correct, even if points are lost. This strategy is not stateful.
```

Comment:
is this last bit true? we still need to store the previous value of the timeseries to detect resets, and also the initial point timestamp for every subsequent point.

Reply:
What I mean is that the __sequence__ of resulting points isn't stateful.  Each point is meaningful on its own, as it gives you a correct average rate over the interval.

I'm trying to contrast it with the true_reset_point strategy.  If you lose the initial "true reset" point, you will interpret subsequent points incorrectly.

Is there different wording I could use to make that clearer.